src/main/scala/nl/knaw/dans/easy/pid/generator/PidFormatter.scala

```diff
@@ -56,16 +58,18 @@ object PidFormatter {
    *                be mapped to chars normally not used with the given radix. The radix
    *                should therefore be sufficiently small to have enough unused chars.
    * @param dashPos position to insert a dash for readability
-   * @param pid     the PID number to format
+   * @param seed    the seed to format
```

Comment:
i'm confused. why format the seed?

Reply:
we need to transform the seed into a PID (either DOI or URN).
`seed` in this function is the seed to transform (format) into a `PID`.
Not sure what is confusing about this?

Reply:
i'm glad you corrected the name of the parameter. 
my point is that the term 'format' seems to imply something more superficial than this transformation.